t/08-autotest.t

```diff
@@ -240,6 +240,18 @@ is(@{$autotest::tests{'tests-fatal'}}{@opts}, @{$autotest::tests{'tests-fatal' .
   && is(@{$autotest::tests{'tests-fatal' . $_}}{name}, 'fatal#' . $_)
   for 1 .. 10;
 
+my ($name, $category) = autotest::parse_test_path("/home/tux/.local/lib/openqa/share/tests/sle/tests/x11/firefox.pm");
+is($name,     'firefox');
+is($category, 'x11');
```

Comment:
How about

```suggestion
my $casedir = '/home/tux/.local/lib/openqa/share/tests/sle/tests';
is(autotest::parse_test_path("$casedir/x11/firefox.pm"), ('firefox', 'x11'));
```

for the sake of brevity?

Reply:
updated patch.